fuzz/fuzz_targets/fuzz_bit_string_with_no_unused_bits.rs

```diff
@@ -0,0 +1,7 @@
+#![no_main]
+use libfuzzer_sys::fuzz_target;
+
+fuzz_target!(|data: &[u8]| {
+    let mut reader = untrusted::Reader::new(untrusted::Input::from(data));
+    let _ = ring::io::der::bit_string_with_no_unused_bits(&mut reader);
+});
```

Comment:
What I would expect here is that the thing inside `fuzz_target!` would be a function that can be shared between fuzz tests and unit known-answer tests.

Also, I want to clarify the goal here: The purpose of this test is to verify that `bit_string_with_no_unused_bits` never panics or crashes, right?